screens/Scan/SuccessfullySharedModal.tsx

```diff
@@ -41,13 +42,38 @@ export const SharingSuccessModal: React.FC<
             {sendVcScreenController.receiverInfo.name}
           </Text>
         </Column>
-        <Column margin="0 0  0">
-          <Button
-            type="gradient"
-            title={t('ScanScreen:status.accepted.gotohome')}
-            onPress={scanLayoutController.DISMISS}
-          />
-        </Column>
+        <Row
```

Comment:
Can we also introduce any testIDs for the newly introduced components here?

Reply:
Done